test/e2e/backup_test.go

```diff
@@ -9,75 +9,55 @@ import (
 	. "github.com/onsi/gomega"
 	"golang.org/x/net/context"
 
-	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/vmware-tanzu/velero/pkg/cmd/cli/install"
 )
 
 var (
-	backupName  string
-	restoreName string
+	uuidgen              uuid.UUID
+	err                  error
+	backupName           string
+	restoreName          string
+	veleroInstallOptions *install.InstallOptions
 )
 
-var _ = Describe("Backup Restore test using Kibishii to generate/verify data", func() {
-
+func veleroInstall(cloudProvider string, useRestic bool) {
+	flag.Parse()
+	Expect(EnsureClusterExists(context.TODO())).To(Succeed(), "Failed to ensure kubernetes cluster exists")
+	Expect(err).To(Succeed(), "Failed to instantiate cluster client")
+	uuidgen, err = uuid.NewRandom()
+	Expect(err).To(Succeed())
+	veleroInstallOptions, err = GetProviderVeleroInstallOptions(cloudProvider, cloudCredentialsFile, bslBucket, bslPrefix, bslConfig, vslConfig, getProviderPlugins(cloudProvider))
+	Expect(err).To(Succeed(), "Failed to get Velero InstallOptions for provider azure")
+	veleroInstallOptions.UseRestic = useRestic
+	Expect(InstallVeleroServer(veleroInstallOptions)).To(Succeed(), "Failed to install Velero on kind cluster")
+}
+
+// Test backup and restore of Kibishi using restic
+var _ = Describe("Velero tests on Kind cluster using cloudProvider for object storage and Restic for volume backups", func() {
```

Comment:
I thought the tags at the beginning of the tests were a good idea. I know these tests aren't provider specific any more but perhaps we could still include the `[Restic]` tag?

Reply:
Yes, I liked "tags" too. I am going to add a `[Restic]` tag to this. Thanks for the suggestion.